contrib/pyln-client/pyln/client/gossmap.py

```diff
@@ -73,7 +73,7 @@ def _parse_features(featurebytes):
     return result
 
 
-class GossipStoreHeader(object):
+class GossipHeader(object):
```

Comment:
But gossip is a class of protocol messages (which contain headers), gossipstore is a file which contains a header and those messages (and others).

Reply:
Or maybe `GossipStoreMessageHeader` to be most precise ;)

Reply:
Now it's `GossipStoreMsgHeader`